pkg/interfaces/contracts/vault/IBatchRouter.sol

```diff
@@ -93,6 +93,57 @@ interface IBatchRouter {
         bytes calldata userData
     ) external payable returns (uint256[] memory pathAmountsIn, address[] memory tokensIn, uint256[] memory amountsIn);
 
+    /**
+     * @notice Add arbitrary amounts of underlying tokens to an ERC4626 pool through the buffer.
+     * @param pool Address of the liquidity pool
```

Comment:
I think we need to define here (and for the other new functions) what an "ERC4626 pool" is. We've lost a bit of flexibility compared to the original "token-based" design, where you could mix ERC4626 and standard tokens in a pool. You can still do that, but these add/remove liquidity functions require that *all* pool tokens are ERC4626, or the wrap/unwrap functions would revert.

`@dev An "ERC4626 pool" is one in which all tokens conform to the IERC4626 yield-bearing token standard (e.g., waDAI).`

Reply:
Thanks!